hazelcast/include/hazelcast/client/ITopic.h

```diff
@@ -50,37 +53,26 @@ namespace hazelcast {
             }
 
             /**
-            * Subscribes to this topic. When someone publishes a message on this topic.
-            * onMessage() function of the given MessageListener is called. More than one message listener can be
-            * added on one instance.
-            * listener class should implement onMessage function like in the following example:
+            * Subscribe to this topic. 
+            * When a message is published on this topic, 
+            * the callback function that was provided to the given MessageListener object is called.
+            * \see MessageListener
             *
-            *      class MyMessageListener {
-            *      public:
-            *          //....
-            *
-            *          void onMessage(topic::Message<std::string> message) {
```

Comment:
needs to be updated

Reply:
will do

Reply:
I removed the example here and haven't written a new one, because from what I observed, this is not a common thing through the whole codebase (addEntryListener, addItemListener don't have examples as comments). And we already have an example under the `examples/` directory. 
As a suggestion for giving examples in the doc, doxygen has a `\example` command which we can use to give references to the examples in the `examples/` directory.

Reply:
`example` tag is a good idea, will it include the examples in the doxygen generated pages?

Reply:
Yes, I guess it does. We need to set EXAMPLE_PATH.